lib/process-services-cloud/src/lib/process/process-list/services/process-list-cloud.service.spec.ts

```diff
@@ -99,4 +106,16 @@ describe('ProcessListCloudService', () => {
             }
         );
     });
+
+    it('should make post request if admin', (done) => {
+        service.isAdmin = true;
+        const processRequest = { appName: 'fakeName', skipCount: 0, maxItems: 20, service: 'fake-service' } as ProcessQueryCloudRequestModel;
+        spyOn(alfrescoApiService, 'getInstance').and.callFake(returnCallOperation);
+        service.getProcessByRequest(processRequest).subscribe((res) => {
```

Comment:
can you adapt this test to use async/await by using toPromise()?
 https://alfresco.atlassian.net/wiki/spaces/TECH/pages/510919005/FE+unit+tests+best+practices#Adapting-a-test-written-with-done()-to-use-promises-instead

Reply:
Sure!